setup.py

```diff
@@ -137,31 +138,38 @@ def run(self):
       author=ABOUT['__author__'],
       author_email=ABOUT['__email__'],
       description=ABOUT['__summary__'],
+      license=ABOUT['__license__'],
       long_description=long_description,
 
       install_requires=INSTALL_REQUIRES,
       extras_require=EXTRAS_REQUIRE,
 
-      packages=[PACKAGE_NAME, 'cli'],
+      packages=find_packages(exclude=["tests"]),
       package_data={PACKAGE_NAME: [
-          'blockchain/eth/*', 'project/contracts/*',
-          'blockchain/eth/sol_source/contracts/lib/*',
-          'blockchain/eth/sol_source/contracts/zepellin/math/*',
-          'blockchain/eth/sol_source/contracts/zepellin/ownership/*',
-          'blockchain/eth/sol_source/contracts/zepellin/token/*']},
+          'network/nicknames/web_colors.json',
+          'blockchain/eth/sol/source/contracts/*',
+          'blockchain/eth/sol/source/contracts/lib/*',
+          'blockchain/eth/sol/source/contracts/proxy/*',
+          'blockchain/eth/sol/source/zepellin/math/*',
+          'blockchain/eth/sol/source/zepellin/ownership/*',
+          'blockchain/eth/sol/source/zepellin/token/*']},
       include_package_data=True,
-      entry_points='[console_scripts]\n{}=cli.main:cli'.format(PACKAGE_NAME),
+
+      entry_points={'console_scripts': ['{0}={0}.cli:cli'.format(PACKAGE_NAME)]},
```

Comment:
Well, again. Would that ever work with non-editable installation?

Reply:
(since it's not a module)

Reply:
To clarify - I think this is acceptable for a developer installation - I went back-and-forth for a while on where to put this file, but ultimately inside `nucypher` is the easiest place to manage it. 

Are you able to pip install the project directory? `pip install ./nucypher`

Reply:
Ah wait, you moved it to nucypher/cli.py. Perfect!

Reply:
Can test like:
```
virtualenv .test -p python3
source .test/bin/activate
pip3 install git+https://github.com/KPrasch/nucypher.git@preversioning
nucypher
```